cmd/wksctl/init/init.go

```diff
@@ -181,10 +217,10 @@ func updateManifests(options initOptionType) error {
 			}
 			return nil
 		})
-	if matches < len(updates) {
+	if matches < 2 {
 		return errors.New("Both 'flux.yaml' and 'wks-controller.yaml' must be present in the repository")
```

Comment:
Need a better condition to show this error message now. as `weave-net.yaml` is optional and would make 3.